### PR TITLE
added : Text-inputs and forms

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Please get me reviewed guys thanks
live-https://loving-shockley-e39c7a.netlify.app

### CODE REVIEW

**Comment by anik31** [comment]
Looks good

**Comment by hsnice16** [comment]
Everything looks good. Nice work, 👍

File: components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.form{
+    border:solid var(--6C757D) ;
```
Comment (sainath7878) [inline components/input/input.css]: Maintain space after :

**Comment by Developers-codz** [comment]
looks good to me.

**Comment by Dev-AkshatGupta** [comment]
> looks good to me.

Thanks for Reviewing

**Comment by Dev-AkshatGupta** [comment]
> Everything looks good. Nice work, 👍

Thanks man for reviewing

**Comment by Dev-AkshatGupta** [comment]
> Looks good

Thanks for reviewing it buddy

File: components/input/input.css (hunk 1 above)
Comment (Dev-AkshatGupta) [reply components/input/input.css]: Ok I will fix the indentation thanks for Reviewing :)

**Pull request merged**